nautobot/core/templates/inc/custom_fields_panel.html

```diff
@@ -17,6 +18,13 @@
                                 <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
                             {% elif field.type == 'multi-select' and value %}
                                 {{ value|join:", " }}
+                            {% elif field.type == 'json' and value %}
+                                <p>
+                                    <button class="btn btn-xs btn-primary" type="button" data-toggle="collapse" data-target="#{{ field.name }}" aria-expanded="false" aria-controls="{{ field.name }}">
```

Comment:
Probably would be safest to still prepend something like `cf_` to the element `id` just in case the user has a custom field with a name like `"main"`, `"advanced"`, or `"navbar"` that matches the id of an existing element in the page templates.